tests/e2e/util.go

```diff
@@ -5647,3 +5646,26 @@ func startCSIPods(ctx context.Context, client clientset.Interface, csiReplicas i
 	isServiceStopped := false
 	return isServiceStopped, err
 }
+
+// waitForStsPodsToBeInRunningState function waits till all the pods comes up
+func waitForStsPodsToBeInReadyRunningState(ctx context.Context, client clientset.Interface, namespace string,
```

Comment:
you don't need `statefulSetReplicaCount` param right? you can just use statefulSets[i].spec.replicas

Reply:
that is correct. Acknowledged.
changes done